tests/models/wav2vec2_with_lm/test_processor_wav2vec2_with_lm.py

```diff
@@ -194,7 +195,14 @@ def test_decoder(self):
         self.assertEqual(decoded_decoder[-2], decoded_processor.logit_score)
         self.assertEqual(decoded_decoder[-1], decoded_processor.lm_score)
 
-    def test_decoder_batch(self):
+    @parameterized.expand(
+        [
+            [None],
+            ["fork"],
+            ["spawn"],
+        ]
+    )
```

Comment:
This can all fit in one line
```suggestion
    @parameterized.expand([[None], ["fork"], ["spawn"]])
```